scripts/actions/fetch-observability-packs.js

```diff
@@ -73,6 +74,7 @@ const fetchPacks = async (queryString, url, token) => {
     const results = await res.json();
 
     if (results.errors) {
+      console.log(results.errors);
```

Comment:
It looks like we're `throw`ing, `catch`ing and calling `console.error` down below so we probably don't need this extra `console.log`.

```suggestion
```